.github/build.yaml.gomplate

```diff
@@ -672,6 +672,31 @@
             {{{ $dir }}}.tar
 {{{end}}}
 
+{{{define "check_labels"}}}
+{{{ $config := (datasource "config") }}}
+    {{{- if has $config "labels" }}}
+  check-labels:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: mheap/github-action-required-labels@v1
+        with:
+          mode: exactly
+          count: {{{ len $config.labels }}}
+          labels: "{{{ join $config.labels  " "}}}"
+    {{{- end }}}
+{{{end}}}
+
+{{{define "local_runner_deps"}}}
+{{{ $config := (datasource "config") }}}
+    {{{- if $config.local_runner }}}
+  local-runner-deps:
+    runs-on: [{{{ join $config.runs_on ", "}}}]
+    steps:
+      - run: |
+          apt-get install sudo
```

Comment:
shouldn't those changes be part of the Docker image instead?

Reply:
probably yes, but currently is faster to test it like this :D

Reply:
I just thought that it would be impossible to apt-get something from the runner user xD my bad

Reply:
I have added sudo to the Dockerfile see PR #615

Reply:
> I just thought that it would be impossible to apt-get something from the runner user xD my bad

it shouldnt but who knows :D

it fails so....